Assets/MixedRealityToolkit-SDK/Profiles/DefaultMixedRealityControllerMappingProfile.asset

```diff
@@ -540,150 +592,150 @@ MonoBehaviour:
       invertXAxis: 0
       invertYAxis: 0
     - id: 1
-      description: Trigger Position
-      axisType: 3
-      inputType: 10
+      description: Spatial Grip
```

Comment:
Have you checked that HoloLens hand position comes through spatial pointer and not spatial grip? If I'm remembering correctly, the Unity API surfaces that info across the grip type, not the pointer type.

Reply:
I assigned the grip pose action to the Spatial Grip on the HL